perceval/backends/core/pagure.py

```diff
@@ -0,0 +1,404 @@
+        }
+        items = super().fetch(category,
+                              filter_classified=filter_classified,
+                              **kwargs)
+
+        return items
+
+    def fetch_items(self, category, **kwargs):
+        """Fetch the items (issues)
+
+        :param category: the category of items to fetch
+        :param kwargs: backend arguments
+
+        :returns: a generator of items
+        """
+        from_date = kwargs['from_date']
+        to_date = kwargs['to_date']
+        items = self.__fetch_issues(from_date, to_date)
+        return items
+
+    @classmethod
+    def has_archiving(cls):
+        """Returns whether it supports archiving items on the fetch process.
+
+        :returns: this backend supports items archive
+        """
+        return True
+
+    @classmethod
+    def has_resuming(cls):
+        """Returns whether it supports to resume the fetch process.
+
+        :returns: this backend supports items resuming
+        """
+        return True
+
+    @staticmethod
+    def metadata_id(item):
+        """Extracts the identifier from a Pagure item."""
+
+        return str(item['id'])
+
+    @staticmethod
+    def metadata_updated_on(item):
+        """Extracts the update time from a Pagure item.
+
+        The timestamp used is extracted from 'last_updated' field.
+        This date is converted to UNIX timestamp format. As Pagure
+        dates are in timestamp format the conversion is straightforward.
+
+        :param item: item generated by the backend
+
+        :returns: a UNIX timestamp
+        """
+        ts = int(item['last_updated'])
+        ts = datetime.fromtimestamp(ts).timestamp()
+
+        return ts
+
+    @staticmethod
+    def metadata_category(item):
+        """Extracts the category from a Pagure item.
+
+        This backend generates one type of item which is
+        'issue'.
+        """
+        category = CATEGORY_ISSUE
+
+        return category
+
+    def _init_client(self, from_archive=False):
+        """Init client"""
+
+        return PagureClient(self.namespace, self.repository, self.api_token,
+                            self.sleep_time, self.max_retries, self.max_items,
+                            self.archive, from_archive, self.ssl_verify)
+
+    def __fetch_issues(self, from_date, to_date):
+        """Fetch the issues
+        :param from_date: starting date from which issues are fetched
+        :param to_date: ending date till which issues are fetched
+
+        :returns: an issue object
+        """
+        issues_groups = self.client.issues(from_date=from_date)
+
+        for raw_issues in issues_groups:
+            issues = json.loads(raw_issues)
+            issues = issues['issues']
+            for issue in issues:
+
+                if int(issue['last_updated']) > str_to_datetime(to_date).timestamp():
+                    return
+
+                yield issue
+
+
+class PagureClient(HttpClient, RateLimitHandler):
```

Comment:
RateLimitHandler can be removed, I don't see any rate limit information in the API and in the response.